FirebaseDatabase/Sources/Core/FRepo.m

```diff
@@ -533,7 +533,7 @@ - (void)getData:(FIRDatabaseQuery *)query
         }];
         return;
     }
-    [self.persistenceManager setQueryActive:querySpec];
+    NSNumber *tag = [self.serverSyncTree registerQuery:[query querySpec]];
```

Comment:
I don't think get queries are tagged on the backend, but it looks like the tag is [only used retrieve the query spec](https://github.com/firebase/firebase-ios-sdk/blob/eb30b6fdd809cbfe3a9eb7f3ccf1dd46326bef2d/FirebaseDatabase/Sources/Core/FSyncTree.m#L384) in `applyTaggedQueryOverwriteAtPath`.